### PR TITLE
Add two regression tests to add benchmark regtest and wav-atm coupling test

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
* A 1 day regression test (1 day run and cold start)  for the IC 2013040100 with the same physics/ICs/set-up as the benchmarks was added.  2013040100 was chosen as it frequently was the most unstable IC for previous bugs and will hopefully help us catch bugs in the future.  This will help us know faster if an update will effect upcoming benchmark runs.  (Issue #46)
* Adds WAVEWATCH III as a component in the coupled model. 
* A 1 day regression tests (1 day run and cold start) for the IC 2013040100 that also has wav-atm two way coupling is added.  It essentially mimics the above test plus wav-atm coupling.  This test will be updated when wav-ocean coupling is added. (Issue #31)  Note this test also requires the removal of the option "norebuild" in compsets/nemscommon.input
* A new baseline was created since the newest fv3atm https://github.com/NOAA-EMC/fv3atm/pull/81  changes answers in the above two tests.   Determining why these answers change will be taken up as part of Issue #39 which is still outstanding. 
* The date suite runs now have 4 more tests for 2013 cases (these are not run as part of the standard regression tests but can be turned on for quick-easy testing of 8 IC for stability testing).
* Several app level files were updated to match the latest of the ufs-weather-model 
* Hera module file was updated to use post/8.0.6

This PR closes issues #31 and #46

### CODE REVIEW

File: parm/input.mom6.nml.IN [hunk 1]
```diff
@@ -158,8 +158,8 @@
        random_clds    = .true.
        trans_trac     = .false.
        cnvcld         = .true.
-       imfshalcnv     = @[IMFSHALCNV]
-       imfdeepcnv     = @[IMFDEEPCNV]
+       imfshalcnv     = 2
```
Comment (DeniseWorthen) [inline parm/input.mom6.nml.IN]: I think we probably want to keep imshalcnv and imfdeepcnv as settable parameters in the input.mom6.nml.IN. 
In the benchmark.nml.IN they should remain fixed values.

Comment (JessicaMeixner-NOAA) [reply parm/input.mom6.nml.IN]: Reverted.  2 is the default value in fv3.input so this should not effect any answers.

**Review by DeniseWorthen — APPROVED** [review]
Approved

**Comment by JessicaMeixner-NOAA** [comment]
Regtests are being re-run after last commit.  Depending on how long the queues for hera are, should be done within an hour or so.  Second reviewer has been added.

**Review by binli2337 — APPROVED** [review]

**Comment by JessicaMeixner-NOAA** [comment]
The log files have been updated to confirm that the last change made no changes to answers.

**Pull request merged**